src/index.css

```
@@ -2,6 +2,22 @@
 @import './styles/themes.css';
 @config "../tailwind.config.js";
 
+@font-face {
+  font-family: 'Orbitron';
```

Stylelint (17.13.0)

[error] Expected no quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)